kuzzleio/kuzzle-admin-console
# Show aggregations in search result if any

## What does this PR do ?

In raw filter mode the user can set aggregations to the search but we didn't show them back in the search result.

### How should this be manually tested?

- Go to raw json search in a collection.
- Enter a filter like this one:

```
{
  "query": {},
  "aggregations": {
    "my_aggs": {
      "terms": {
        "field": "foo"
      }
    }
  }
}
```

- Hit Search

In the result list you should see the `Content`, `Meta` and newly added `Aggregations` inside the documents.

## Preview

https://deploy-preview-528--kuzzle-backoffice-last.netlify.com/
